src/inference/src/ie_core.cpp

```diff
@@ -1024,7 +1053,7 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                         "get_property is also possible for the individual devices before creating the BATCH on top.");
 
         if (device_name.empty()) {
-            return get_property_for_core(name);
+            return coreConfig.get_core_config(name);
```

Comment:
can you share an example, how to get the coreConfig in plugin from ICore

Reply:
CoreConfig is internal helper class, which can be used to cache global properties that can be applied for all plugins or specified plugin. It is in CoreImpl level, not ICore level. Does plugin need access this data?

Reply:
maybe in some scenarios, like in auto, we may need to deduce optimal infer requests before actual network loaded, the  number will vary much when batching enable and disabled

Reply:
sure, new commit has been submitted to consider AUTO plugin's requirement.